go/service/deferred_deep_links.go

```diff
@@ -39,48 +36,21 @@ func (c installReferrerHandler) CallbackWithString(s string) {
 		}
 	}()
 
-	// TODO remove this log line before the feature is released: invite links
-	// are sensitive and we do not want to log them.
-	m.Warning("installReferrerHandler InstallReferrerListener got %v", s)
-
-	// This could be expanded to all sorts of deferred deep links, not just
-	// invite links.
-	inviteKey, found, err := parseInstallReferrer(s)
-	if err != nil {
-		m.Warning("Error in parseInstallReferrer: %v", err)
-		return
-	}
-	if !found {
-		m.Debug("installReferrerHandler#CallbackWithString: invite not found")
-		return
-	}
+	// Play Store seems to replace our URL-encoded plus sign with a space, so we
+	// need to work around it.
+	s = strings.Replace(s, " ", "+", -1)
```

Comment:
It seems + can also be used to encode spaces, which is why the play store is decoding it as a space. I think the fix here is to change the way the website encodes this. Can you check if this is possible? 
More info here: https://developers.google.com/maps/documentation/urls/url-encoding

Reply:
>For example, 上海+中國 in UTF-8 would be URL-encoded as %E4%B8%8A%E6%B5%B7%2B%E4%B8%AD%E5%9C%8B
`https://play.google.com/store/apps/details?id=io.keybase.ossifrage&referrer=keybase%3A%2F%2Fteam-invite-link%2F5b81a5512e0ef2e9c4afe71921c2e727%2Fe6afb2k%2Bgx2ypqqv9mw9kpebhhhy`
If we run into any issues with this code, we'll just run it only on crypto invite links.

Reply:
Maybe we are decoding twice somehow? 
I agree the scope of this code is limited, but it seems weird that the playstore has such a weird decoding bug.